_posts/2020-03-30-exploting-spatial-diversity-using-unetstack3.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+comments: true
+title: Exploiting Dsitributed Spatial Diversity Using UnetStack
+date: 30/03/2020
+author: Prasad Anjangi
+categories: howto
+feature-img: "assets/img/sd-unet.jpg"
+thumbnail: "assets/img/sd-unet.jpg"
+tags: [howto, spatial diversity, cooperative diversity, robustness, data rate, performance boost]
```

Comment:
- "cooperative communication" rather than "cooperative diversity"?

We might want to ratify a set of tags that we should use across all blogs.

Reply:
Fixed.